stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Attachment.kt

```diff
@@ -24,12 +24,19 @@ public data class Attachment(
     var mimeType: String? = null,
     @SerializedName("file_size")
     var fileSize: Int = 0,
+    @SerializedName("title")
     var title: String? = null,
+    @SerializedName("text")
     var text: String? = null,
+    @SerializedName("type")
     var type: String? = null,
+    @SerializedName("image")
     var image: String? = null,
+    @SerializedName("url")
     var url: String? = null,
+    @SerializedName("name")
     var name: String? = null,
+    @SerializedName("fallback")
```

Comment:
`@SerializedName ` so proguard doesn't kill us

Reply:
We have a [`consumer-proguard-rules.pro`](https://github.com/GetStream/stream-chat-android/blob/master/stream-chat-android-client/consumer-proguard-rules.pro) file, I think that should be enough to take care of this. (Not sure if it's up-to-date, Studio shows some errors in it on `develop` 😬)

Reply:
`@SerializedName` can be an extra prevention. If we move the class to another package (outside `io.getstream.chat.android.client.api.models`) we will trigger error related to proguard, if serialized name, we can move classes around. I've seen this exact error once, it took me quite a long time to find out that the problem was related to a model class being moved to a package that wasn't covered by the proguard rules.

Reply:
Alright, I'm not bothered by it too much, it'll be nuked when we manage to remove Gson anyway 😄